Client/src/components/Shared/Navbar/Navbar.jsx

```
@@ -1,41 +1,137 @@
+            </Navbar.Link>
+            <Navbar.Link href="/services" className="md:px-4">
+              Videos
+            </Navbar.Link>
+          </Navbar.Collapse>
+
+          {/* Right Section - User Profile & Actions */}
+          <div className="order-2 md:order-4 flex items-center gap-3 flex-shrink-0 ml-auto md:ml-0">
+            {user ? (
+              <>
+                {/* Notification Bell */}
+                <button className="p-2.5 hover:bg-gray-100 rounded-full dark:hover:bg-gray-700 transition-colors duration-200">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)